### DISCUSSION TITLE
Poll Reset

### DISCUSSION THREAD

**Discussion by Blitzbirnep** [ClaperCo/Claper#60] [General]
Hi, I was wondering if resetting a poll without deleting it and retyping all the questions and answers is possible.

My use case would be to test the setup before the presentation begins and than once it started reset all polls which I have tested to the initial state.

Btw. Awesome work!

**Comment by alxlion** [comment]
Hey, you can create another presentation with the same file and import interactions (polls, forms…) from another presentation without the answers. 

![CleanShot 2023-10-12 at 14 12 33](https://github.com/ClaperCo/Claper/assets/1835726/0a2a61c8-8f04-4025-95b1-b9a9ed8ce1ac)
